add submenu for changing completion status to game context menu (#1796)

V0idhead opened the pull request:
Since this is my first contribution I wanted to make sure I can get everything to work before making promises on the ticket. So sorry for the "surprise" pull request.

As proposed in https://github.com/JosefNemec/Playnite/issues/1796 I added an item in the game context menu to change the completion status:
![completionStatus](https://user-images.githubusercontent.com/65420350/82093319-2f22f800-96fb-11ea-858f-f232a146b57a.png)

If this change is welcome I'd also make a similar change to the version of the context menu that handles multiple selected games.

Obviously the Header should be "Set Completion Status", but I don't know the process for updating the localizations so I reused the existing string.

I have verified that:

* [x]  These changes work, by building the application and testing them.
* [x]  Pull request is targeting `devel` branch.
* [ ]  I added myself into [contributors file](https://github.com/JosefNemec/Playnite/blob/devel/source/Playnite.DesktopApp/Resources/contributors.txt) if I want to receive contribution credit in the application itself.

JosefNemec commented:
To add a new translatable string, create new entry in `LOCSource.xaml` file.

Also, please add check list from the PR template.

JosefNemec commented on source/Playnite/GamesEditor.cs:
```diff
@@ -328,6 +328,12 @@ public void ToggleFavoriteGame(List<Game> games)
 
+        public void SetCompletionStatus(Game game, CompletionStatus status)
+        {
+            game.CompletionStatus = status;
```
Can you add check if new status is actually different and skip game update if it's the same?

V0idhead commented:
Sure,

 - I added an entry LOCSource.xaml and also added an entry to the german version
 - added the checklist
 - made sure the new status is different before updating the game
 - also added the menu to the version of the context-menu that handles multiple games
